pipelines/rj_iplanrio__taxirio/README.md

```
@@ -0,0 +1,202 @@
+- **metricsdriverunoccupieds**: Métricas de motoristas desocupados
+
+### Dumps Semanais (Overwrite)
+- **users**: Usuários do sistema
+
+### Dumps Diários (Append)
+- **races**: Corridas realizadas (particionado por ano/mês/dia)
+- **passengers**: Passageiros (particionado por ano/mês/dia)
+
+## Estrutura do Projeto
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)